scripts/validate_cclw_sheet/validate-ingest.sh

```diff
@@ -12,36 +12,36 @@ PASSWORD=${SUPERUSER_PASSWORD:-password}
 # ---------- Functions ----------
 
 wait_for_server() {
-    printf 'Waiting for server.'
-    until $(curl --output /dev/null --silent  ${TEST_HOST}/health); do
-        printf '.'
-        sleep 1
-    done
-    echo
+	printf 'Waiting for server.'
+	until $(curl --output /dev/null --silent "${TEST_HOST}"/health); do
+		printf '.'
+		sleep 1
+	done
+	echo
 }
 
 get_token() {
-    curl -s \
-    -H "Content-Type: application/x-www-form-urlencoded" \
-    -d "username=${USER}&password=${PASSWORD}" \
-        ${TEST_HOST}/api/tokens | \
-        jq ".access_token" | tr -d '"'
+	curl -s \
+		-H "Content-Type: application/x-www-form-urlencoded" \
+		-d "username=${USER}&password=${PASSWORD}" \
+		"${TEST_HOST}"/api/tokens |
+		jq ".access_token" | tr -d '"'
 }
 
 validate_csv() {
-    TOKEN=$(get_token)
-    curl -s \
-        -H "Authorization: Bearer ${TOKEN}" \
-        -F "law_policy_csv=@${CSV_FILE}" \
-        ${TEST_HOST}/api/v1/admin/bulk-ingest/validate/cclw
+	TOKEN=$(get_token)
+	curl -s \
+		-H "Authorization: Bearer ${TOKEN}" \
+		-F "law_policy_csv=@${CSV_FILE}" \
+		"${TEST_HOST}"/api/v1/admin/bulk-ingest/validate/cclw
 }
 
 echo "Validating as ${USER}"
 wait_for_server
 
 echo
 echo "👉👉👉  Validate CSV"
-validate_csv > validation.json
+validate_csv >validation.json
```

Comment:
Interesting!

Reply:
This removes the ambiguity of the space being a part of the filename